### ISSUE TITLE
Package loading order and crash without 'standard-app-packages'

### ISSUE THREAD

**Issue by yeputons** [sdarnell/meteor-win#31]
Here's a demo: https://github.com/yeputons/meteor-package-load-and-fail . I've got this issue while working on Meteorite (oortcloud/meteorite#245). If you run this application on Linux, you will get:

```
Test package 1 installed (branch/master)
Test package 2 installed (branch/master)
Program has no main() function.
```

But if you run it on Windows, if fails after the first line:

```
Test package 1 installed (branch/master)
Program has no main() function.
```

If you add `standard-app-packages`, it will work OK and similar both on Windows and Linux.

**Comment by sdarnell** [comment]
I've had an initial look, and I think there is some form of crash going on, possibly in the fibers code.
It needs more investigation, but I can illustrate part of the issue by adding an extra console.log() line before the one you have in package 1. When I do this, I get that line but not the original trace.

Specifically, if I change the server.js to be:

```
console.log('YYY1');
console.log('Test package 1 installed (branch/master)');
```

I just get the first line output on the console:

```
=> Meteor server running on: http://localhost:3000/
I20140129-21:44:28.966(0)? YYY1
W20140129-21:44:29.307(0)? (STDERR) Program has no main() function.
=> Exited with code: 1
```

So clearly it isn't quite as simple as just a package load order issue (and I checked that the bundler seems to have loaded both packages just fine).

I'll have to dig a little further, but I have seen some issues with fibers on windows where output just disappears when an error occurs.

**Comment by yeputons** [comment]
@sdarnell may be a buffering issue. Console writes are buffered in node on Windows, but they are not on Linux. I've seen a very old issue about it in node's repository.

**Comment by sdarnell** [comment]
@yeputons I looked at the exit npm package and I felt a little uneasy. I did some more exploring and after investigating a few different options, I finally came up with a solution that I was comfortable with. See change a309aca which is probably a bit overkill for normal usage, but I wanted to support any potential usage.

I think the key issue is that Windows pipes are documented as non-blocking and I think that libuv code is invoked when executing a child process and capturing the output (as happens with 'meteor run').

With this change the example repo you gave works the same as on Linux.
Apologies for the delay.

As 0.7.1 is about to be released, I'll roll it into that release.
